solcore/interpolate.py

```diff
@@ -259,7 +260,8 @@ def _check_bounds(self, x_new):
 class BilinearInterpolation(object):
     """docstring for BilinearInterpolation"""
 
-    def __init__(self, x=None, y=None, z=None, fill_value=0.):
+    def __init__(self, x: list = None, y: list = None,
+                 z: list = None, fill_value: float = 0.):
```

Comment:
Actually, this is an error (not yours! It was already there). x, y and z cannot be `None` and that default value must be removed. 

Keeping that aside, the type of those three should be `Union[list, np.ndarray]`